plugins/fsm/src/plugin.ts

```diff
@@ -11,7 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import type { Ref } from '@anticrm/core'
+import isEqual from 'lodash/isEqual'
```

Comment:
Could you use: fast-equals instead, it is faster and already used.

Reply:
Done